frontend/components/MobileAccordion.vue

```diff
@@ -0,0 +1,89 @@
+<template>
+  <div class="p-4">
+    <!-- Mobile View -->
+    <div class="md:hidden">
+      <div
+        v-for="(item, menuKey) in menuItems"
+        :key="menuKey"
+        class="border-b-2 border-gray-200 py-2"
+      >
+        <button
+          class="w-full flex justify-between items-center"
+          @click="toggleItem(item.id)"
+        >
+          <span class="font-bold italic text-offblack">{{ item.title }}</span>
+          <span class="text-xl px-2">
+            {{ isActive(item.id) ? '-' : '+' }}
+          </span>
+        </button>
+        <transition name="fade">
+          <div v-if="isActive(item.id)" class="mt-2">
+            <MultiColumnContent :sections="item.sections" />
+          </div>
+        </transition>
+      </div>
+    </div>
+    <!-- Desktop View -->
+    <div class="hidden md:block">
```

Comment:
I know I might have mislead you with talking about classes like `hidden md:block` and vice versa for mobile / desktop, but that was not intended for the content in the context of accordion but button.

Reply:
my new version uses one component reference for mobile and updated